etc/taskcluster/windows/bootstrap.ps1

```diff
@@ -12,7 +12,7 @@
 
 # Optional
 $client.DownloadFile(
-    "http://download.tuxfamily.org/dvorak/windows/bepo.exe",
+    "https://download.tuxfamily.org/dvorak/windows/bepo.exe",
```

Comment:
@SimonSapin Btw, did this even work before? $client is created just a few lines below. Shouldn't this be placed after the Expand-ZIPFile function to have a working client and to be restricted to TLS 1.2?

Reply:
It’s quite possible that I’ve moved things around after the last time I ran this script, and this doesn’t work as-is.

This script is only ever used when building a new AMI, which I expect we’ll do a few times a year at most. https://github.com/servo/servo/tree/master/etc/taskcluster/windows#readme

As far as I understand the `SecurityProtocol` line doesn’t restrict to TLS 1.2, it only enables it. (This is for servers like github that disable older TLS versions.)

Reply:
(I fear I wouldn't be able to fix and verify such an off-topic bug.)
As the only intention of this PR is to switch to https and not to regress anything I moved this download a bit down to make sure it's covered by the TLS policy as well.